<launch>
  <!-- turtlebot_teleop_key already has its own built in velocity smoother -->
  <node pkg="saetta" type="saetta_base" name="saetta_teleop_keyboard"  output="screen">
    <param name="scale_linear" value="0.5" type="double"/>
    <param name="scale_angular" value="1.5" type="double"/>
    <remap from="saetta_teleop_keyboard/cmd_vel" to="/saetta/velocity"/>
  </node>
</launch>
